ee_site/static/src/js/dynamic_fill_image.js

```diff
@@ -0,0 +1,54 @@
+// lazy image loader based on
+// https://developers.google.com/web/fundamentals/performance/lazy-loading-guidance/images-and-video/#inline_images
+(function () {
```

Comment:
Czy ten śmieszny nieczytelny boilerplate z definiowaniem funkcji i odpalaniem jej w tym samym momencie jest na pewno nadal potrzebny? On służył do oddzielenia kodu we własnym wydzielonym namespace, ale jeśli teraz to są importowane moduły to czy to się nie dzieje samo?

Reply:
racja, to niepotrzebne